standard/classes.md

```diff
@@ -412,23 +413,31 @@ Each *type_parameter_constraints_clause* consists of the token `where`, followed
 
 The list of constraints given in a `where` clause can include any of the following components, in this order: a single primary constraint, one or more secondary constraints, and the constructor constraint, `new()`.
 
-A primary constraint can be a class type or the ***reference type constraint*** `class` or the ***value type constraint*** `struct`. A secondary constraint can be a *type_parameter* or *interface_type*.
+A primary constraint can be a class type, the ***reference type constraint*** `class`, the ***value type constraint*** `struct`, or the ***unmanaged type constraint*** `unmanaged`. 
+
+A secondary constraint can be a *type_parameter* or *interface_type*.
 
 The reference type constraint specifies that a type argument used for the type parameter shall be a reference type. All class types, interface types, delegate types, array types, and type parameters known to be a reference type (as defined below) satisfy this constraint.
 
 The value type constraint specifies that a type argument used for the type parameter shall be a non-nullable value type. All non-nullable struct types, enum types, and type parameters having the value type constraint satisfy this constraint. Note that although classified as a value type, a nullable value type ([§8.3.11](types.md#8311-nullable-value-types)) does not satisfy the value type constraint. A type parameter having the value type constraint shall not also have the *constructor_constraint*, although it may be used as a type argument for another type parameter with a *constructor_constraint*.
 
 > *Note*: The `System.Nullable<T>` type specifies the non-nullable value type constraint for `T`. Thus, recursively constructed types of the forms `T??` and `Nullable<Nullable<T>>` are prohibited. *end note*
 
-Pointer types are never allowed to be type arguments and are not considered to satisfy either the reference type or value type constraints.
+The token `unmanaged` is neither a keyword nor a contextual keyword. When it is encountered, it will either:
+- Bind to a type named `unmanaged`
+- Bind to no type, in which case, it is interpreted as the unmanaged type constraint.
+
+The unmanaged type constraint specifies that a type argument used for the type parameter shall be a non-nullable unmanaged type.
+
+Pointer types are never allowed to be type arguments and are not considered to satisfy either the `reference` type or `value` type constraints.
```

Comment:
`reference` and `value` markup seems inappropriate here because, unlike `unmanaged`, those words do not have specific meanings in C# source code.

Reply:
I think those are both typos, also doesn't `unmanaged` need to be added? So we get:

> Pointer types are never allowed to be type arguments and are not considered to satisfy either the `class` type, `struct`, or `unmanaged` type constraints.

Reply:
I agree that it should say `class` and `struct`, but as pointers are unmanaged types, Nigel's suggestion reads a little oddly. How about something like:

> Pointer types are never allowed to be type arguments, and don't satisfy any type constraints, even `unmanaged`, despite being unmanaged types.

Reply:
On the TG2 call of 2023-02-01, we agreed with this approach.